Formula/audacious.rb

```diff
@@ -60,6 +60,8 @@ class Audacious < Formula
   depends_on "sdl2"
   depends_on "wavpack"
 
+  uses_from_macos "curl"
```

Comment:
Can we just disable it instead?

Reply:
Not without modifying files https://github.com/audacious-media-player/audacious-plugins/blob/audacious-plugins-4.2/src/scrobbler2/meson.build#L1-L5
```meson
curl_dep = dependency('libcurl', version: '>= 7.9.7', required: false)
have_scrobbler2 = curl_dep.found() and xml_dep.found()


if have_scrobbler2
```

As long as `curl` is installed in build environment, it will be used.